### PR TITLE
added TestFairy integration

### PR DESCRIPTION
**Pull request by bouhady**
Remove unused fragments from sideMenu

### CODE REVIEW

File: app/src/debug/java/com/startach/yedidim/testfairy/TestFairyInit.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import com.testfairy.TestFairy;
+
+/**
+ * Created by yb34982 on 24/12/2017.
+ */
+
+public class TestFairyInit {
+    public static void start(Activity activity) {
+        TestFairy.begin(activity, "6f4c3da555567854de9cc48b9844007c2c63eeb2");
```
Comment (yosriz) [inline app/src/debug/java/com/startach/yedidim/testfairy/TestFairyInit.java]: shouldn't we hide the TestFairy id/secret?

File: app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java [hunk 2]
```diff
@@ -1,44 +1,61 @@
...
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import timber.log.Timber;
 
 public class MainPageActivity extends AppCompatActivity
         implements NavigationView.OnNavigationItemSelectedListener {
     // TODO: Refactor to Factory
-    private Fragment personalFragment = new PersonalInfoFragment();
+//    private Fragment personalFragment = new PersonalInfoFragment();
```
Comment (yosriz) [inline app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java]: is it intentional commented out and not deleted?

File: app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java [hunk 3]
```diff
@@ -76,18 +104,19 @@ public boolean onNavigationItemSelected(MenuItem item) {
...
         int id = item.getItemId();
 
-        if (id == R.id.nav_main_screen) {
-            setTitle(R.string.title_nav_main_page);
-            startFragment(dispatcherFragment);
-        } else if (id == R.id.nav_personal_info) {
-            setTitle(R.string.title_nav_personal_information);
-            startFragment(personalFragment);
-        } else if (id == R.id.nav_settings) {
+//        if (id == R.id.nav_main_screen) {
```
Comment (yosriz) [inline app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java]: same, why not to delete?

File: app/src/main/java/com/startach/yedidim/Activity/SplashScreenActivity.java [hunk 4]
```diff
@@ -24,6 +25,7 @@
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
+        TestFairyInit.start(this);
```
Comment (yosriz) [inline app/src/main/java/com/startach/yedidim/Activity/SplashScreenActivity.java]: should be done on application, there're cases where app can be launched from notification directly to event info activity, (maybe EventInfoActivity should be fragment and part of the main activity? still not sure the splash screen will be displayed there)

File: app/src/main/res/menu/activity_main_page_drawer.xml [hunk 5]
```diff
@@ -2,19 +2,19 @@
 <menu xmlns:android="http://schemas.android.com/apk/res/android">
 
     <group android:checkableBehavior="single">
-        <item
-            android:id="@+id/nav_main_screen"
-            android:icon="@drawable/ic_action_home"
-            android:title="@string/title_nav_main_page"
-            android:checked="true"/>
+        <!--<item-->
```
Comment (yosriz) [inline app/src/main/res/menu/activity_main_page_drawer.xml]: delete?

File: app/src/debug/java/com/startach/yedidim/testfairy/TestFairyInit.java (hunk 1 above)
Comment (bouhady) [reply app/src/debug/java/com/startach/yedidim/testfairy/TestFairyInit.java]: No need.

File: app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java (hunk 2 above)
Comment (bouhady) [reply app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java]: When we will know who will update user info we willl deside if removing it

File: app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java (hunk 3 above)
Comment (bouhady) [reply app/src/main/java/com/startach/yedidim/Activity/MainPageActivity.java]: I'll remove

File: app/src/main/res/menu/activity_main_page_drawer.xml (hunk 5 above)
Comment (bouhady) [reply app/src/main/res/menu/activity_main_page_drawer.xml]: yep

**Pull request merged**